src/main/scala/com/codacy/analysis/cli/tools/Tool.scala

```diff
@@ -132,9 +132,20 @@ class Tool(private val plugin: IDockerPlugin) {
 
 object Tool {
 
-  private val allTools = PluginHelper.dockerPlugins.++(PluginHelper.dockerEnterprisePlugins)
+  val allEnterpriseToolShortNames: Set[String] =
+    PluginHelper.dockerEnterprisePlugins.map(_.shortName)(collection.breakOut)
 
-  def allToolShortNames: Set[String] = allTools.map(_.shortName)(collection.breakOut)
+  val allToolShortNames: Set[String] = allEnterpriseToolShortNames ++ PluginHelper.dockerPlugins.map(_.shortName)
+}
+
+class ToolCollector(allowNetwork: Boolean) {
+
+  private val tools =
```

Comment:
This can easily clash with names inside functions or lead to mistakes for other reasons. I think we should rename. Maybe "availableTools" ?